wire-ios-data-model/WireDataModel.xcodeproj/project.pbxproj

```diff
@@ -4858,7 +4858,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
-				ENABLE_MODULE_VERIFIER = YES;
+				ENABLE_MODULE_VERIFIER = NO;
```

Comment:
Since `NO` is the default, we can just remove this line, right?

Reply:
True, will update. Thanks1